cypress/e2e/content/content.spec.js

```diff
@@ -96,28 +96,22 @@ describe("Content Specs", () => {
     });
 
     it("Date Field", () => {
-      cy.get("#12-63ab04-0nkwcc input").click();
+      cy.get("#12-63ab04-0nkwcc button").click();
 
-      cy.get(
-        '.flatpickr-calendar:not(.hasTime) [aria-label="March 5, 2019"]'
-      ).click();
+      cy.get("[role=dialog]").find('[aria-label="Mar 5, 2019"]').click();
 
-      cy.get("#12-63ab04-0nkwcc input").should("have.value", "2019-03-05");
+      cy.get("#12-63ab04-0nkwcc input").should("have.value", "03/05/2019");
     });
 
     it("Date & Time Field", () => {
-      cy.get("#12-f3db44-c8kt0q input.form-control").click();
+      cy.get("#12-f3db44-c8kt0q button").click();
 
-      cy.get(
-        '.flatpickr-calendar.hasTime [aria-label="March 5, 2019"]'
-      ).click();
+      cy.get("[role=dialog]").find('[aria-label="Mar 5, 2019"]').click();
 
-      cy.get(".flatpickr-confirm").click();
-
-      // TODO get aria-label and capture in variable
-      // cy.get("#12-f3db44-c8kt0q .flatpickr-input.form-control.input").click();
-
-      // TODO compare aria-label value with input value
+      cy.get("#12-f3db44-c8kt0q  input").should(
+        "have.value",
+        "03/05/2019 02:30 pm"
```

Comment:
It looks like the previous test was incomplete and did not have an expectation on the value defined. As mentioned in the prior comment format is extremely important. We need to determine the current format date/time values are sent and persisted at the API. We should then ensure the UI maintains that format and the test verifies the format.

Reply:
Pulled from the [source code formatting string](https://github.com/zesty-io/manager-ui/pull/1214/files#diff-f46e1bada931bfb3d22a8c7e5b48da51563447b4b5576e8f19489a1002477620R865) the format should be `YYYY-MM-DD HH:mm:ss`